modules/Kconfig.mcuboot

```diff
@@ -77,3 +77,9 @@ config NRF53_RAM_FLASH_SIZE
 	  is read from external flash, and buffered in  RAM. This option is used
 	  to define forward the size of this RAM region to the partition manager
 	  configuration.
+
+config MCUBOOT_TFM_PRESENT
+	bool "TF-M is present"
+	help
+	  This option allows MCUBOOT perform configurations to allow
```

Comment:
Wording seems a bit off.

Reply:
Maybe something like "This option provides MCUBoot image awareness of TF-M which enables configurations for compatibility between TF-M and MCUBoot."

Reply:
Adapted Sigvart's suggestion ;)